src/dotnet/UI.Blazor/Components/GradientCircle/gradient-circle.css

```diff
@@ -0,0 +1,65 @@
+.background {
```

Comment:
Why this file is named "gradient-circle", btw? Maybe rename it for consistency too?